.github/workflows/integration_test_manta.yml

```diff
@@ -0,0 +1,569 @@
+          path: ${{ github.workspace }}/polkadot-launch-for-${{ matrix.chain-spec.id }}-stderr.log
+      - if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: measure-block-time-rococo-relay-for-${{ matrix.chain-spec.id }}-stdout.log
+          path: ${{ github.workspace }}/measure-block-time-rococo-relay-for-${{ matrix.chain-spec.id }}-stdout.log
+      - if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: measure-block-time-rococo-relay-for-${{ matrix.chain-spec.id }}-stderr.log
+          path: ${{ github.workspace }}/measure-block-time-rococo-relay-for-${{ matrix.chain-spec.id }}-stderr.log
+      - if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: measure-block-time-${{ matrix.chain-spec.id }}-stdout.log
+          path: ${{ github.workspace }}/measure-block-time-${{ matrix.chain-spec.id }}-stdout.log
+      - if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: measure-block-time-${{ matrix.chain-spec.id }}-stderr.log
+          path: ${{ github.workspace }}/measure-block-time-${{ matrix.chain-spec.id }}-stderr.log
+      - if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: rococo-alice-relay-for-${{ matrix.chain-spec.id }}.log
+          path: ${{ github.workspace }}/polkadot-launch/alice.log
+      - if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: rococo-bob-relay-for-${{ matrix.chain-spec.id }}.log
+          path: ${{ github.workspace }}/polkadot-launch/bob.log
+      - if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.chain-spec.id }}-alice.log
+          path: ${{ github.workspace }}/polkadot-launch/9921.log
+      - if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.chain-spec.id }}-bob.log
+          path: ${{ github.workspace }}/polkadot-launch/9922.log
+      - name: parse rococo block times
+        run: |
+          grep '#.*' ${{ github.workspace }}/measure-block-time-rococo-relay-for-${{ matrix.chain-spec.id }}-stdout.log | while read -r line; do words=($line); echo ${words[6]},${words[10]} | tee ${{ github.workspace }}/block-time-rococo.csv; done
+          if [ ! -f ${{ github.workspace }}/block-time-rococo.csv ]; then echo "block times not detected"; exit 1; fi
+          jq -s -R '[split("\n") | .[] | select(length > 0) | split(",") | {block:.[0]|tonumber, time:.[1]|tonumber} ]' ${{ github.workspace }}/block-time-rococo.csv > ${{ github.workspace }}/block-time-rococo.json
+      - if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: block-time-rococo-relay-for-${{ matrix.chain-spec.id }}.json
+          path: ${{ github.workspace }}/block-time-rococo.json
+      # - if: always()
+      # - name: parse manta block times
```

Comment:
do these need to stay commented out?

Reply:
yeah we gotta fix them separetely, but i'd rather move these to TS tests, since the parsing is quite flaky.